Makefile

```diff
@@ -368,13 +368,14 @@ packagemanifests-push:
 
 # special section for cross compilation
 docker-build-arch:
-	docker build -t $(DOCKER_REPO):$(TAG)-$(GOARCH) \
-			--build-arg ARCH=$(GOARCH) \
+	export DOCKER_CLI_EXPERIMENTAL=enabled ;\
+	docker buildx build -t $(DOCKER_REPO):$(TAG)-$(GOARCH) \
+			--platform=linux/$(GOARCH) \
 			--build-arg base_image=$(ALPINE_IMAGE) \
 			-f Docker-multiarch .
 
 package-arch:
-	$(GOBUILD) -o bin/manager-$(GOARCH) main.go
+	CGO_ENABLED=0 GOOS=${GOOS} GOARCH=${GOARCH} $(GOBUILD) -o bin/manager-$(GOARCH) main.go
```

Comment:
![image](https://github.com/VictoriaMetrics/operator/assets/39937150/58de2507-46ea-4b0e-9de9-7a5ab7ae4d0b)
`CGO_ENABLED=0 GOOS=${GOOS} GOARCH=${GOARCH}` is missing